src/main/java/org/mockito/session/MockitoSessionBuilder.java

```diff
@@ -30,17 +30,47 @@
      * Migrate from {@link MockitoAnnotations#initMocks(Object)}
      * to {@link MockitoSession}!
      * <p>
+     * This method may be called multiple times to add multiple, e.g. nested, test class instances.
+     * <p>
      * See code sample in {@link MockitoSession}.
      *
      * @param testClassInstance test class instance that contains fields with Mockito annotations to be initialized.
-     *  Passing {@code null} is permitted and will make the session use a default value.
-     *  The current default is '{@code new Object()}'.
+     *  Passing {@code null} is permitted but will be ignored.
      * @return the same builder instance for fluent configuration of {@code MockitoSession}.
      * @since 2.7.0
      */
     @Incubating
     MockitoSessionBuilder initMocks(Object testClassInstance);
 
+    /**
+     * Adds the test class instances for initialization of fields annotated with Mockito annotations
+     * like {@link org.mockito.Mock}.
+     *
+     * @param testClassInstances test class instances that contains fields with Mockito annotations to be initialized.
+     *  Passing {@code null} or an empty array is permitted but will be ignored.
+     * @return the same builder instance for fluent configuration of {@code MockitoSession}.
+     * @see #initMocks(Object)
+     * @since 2.13.4
+     */
+    @Incubating
+    MockitoSessionBuilder initMocks(Object... testClassInstances);
```

Comment:
Can you explain in what circumstances we pass multiple test instances? Can we recommend to pass single instance and explain that multiple instances are for advanced framework integrations, for example...

Reply:
Will do!

Reply:
Q: Should we throw an IAE in case:
 * an empty vararg/array is passed
 * an array with `null` elements is passed
 * `(Object[])null` is passed